src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PushCommand.cs

```diff
@@ -36,6 +36,11 @@ public static void Register(CommandLineApplication app, Func<ILogger> getLogger)
                     Strings.SymbolSource_Description,
                     CommandOptionType.SingleValue);
 
+                var configurationFile = push.Option(
+                    "--config-file <source>",
```

Comment:
```suggestion
                    "--config-file <file>",
```

In NuGet vocabulary, a source is a feed of packages. I don't know if this was a copy-paste issue, but source really doesn't feel right to me. I don't know if we should say `path` instead of `file`. 

`dotnet restore -h` says `--configile <FILE>`, while `dotnet list package -h` says `--config <CONFIG_FILE>`.  Clear as mud. As consistent as flipping a coin.

Reply:
Just fyi `--configfile <File>` is used for [dotnet nuget trust](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-trust#options-2).